ui/lib/apps/SlowQuery/translations/en.yaml

```diff
@@ -60,8 +60,14 @@ slow_query:
     txn_start_ts_tooltip: Transaction start timestamp, a.k.a. Transaction ID
     write_keys: Write Keys
     write_size: Write Size
-    prewrite_regions: Prewrite Regions
+    prewrite_region: Prewrite Region
```

Comment:
In slow query table, this column names 'Prewrite_region', not 'Prewrite_regions'.

Reply:
As a human readable description maybe it should be "regions" instead of "region" here.